javascript/index.js

```diff
@@ -113,7 +113,7 @@ demo.innerHTML = portfolioCards.reduce((output, portfolio) => (
 
 // Modal popup
 
-const closeButton = document.querySelector('#modal-close');
+const closeButton1 = document.querySelector('#modal-close');
 const btn = document.querySelectorAll('[data-open]');
 const overlay = document.querySelector('.popup-overlay');
 const modal1 = document.querySelector('#modal1');
```

Comment:
- [ ] Please stick to the Figma design your `popup` is not the same as the Figma design kindly fix it check below :point_down:  the comparison between your design and the Figma design :slightly_smiling_face: 
- [ ] Also, check your buttons it's not the same as the Figma design if you check your button you will see the spacing between both buttons is not the same as the Figma design also if you check your button `padding` and `margin` is not same as Figma design so kindly it  :+1: 

| Your Design   |      Figma Design      |
|---------- |:----------------:|
| ![Screenshot from 2023-04-13 13-00-39](https://user-images.githubusercontent.com/45226848/231709173-c40e9826-13e6-4c69-9793-90c04c01e7bf.png) |  ![Screenshot from 2023-04-13 13-04-16](https://user-images.githubusercontent.com/45226848/231709272-55f3d5a9-30ff-42e7-840b-c550afca5377.png)    |
| ![Screenshot from 2023-04-13 13-00-50](https://user-images.githubusercontent.com/45226848/231709330-ea1c46e8-cc71-4cf6-ac49-b0ede7a29043.png)  |  ![Screenshot from 2023-04-13 13-04-31](https://user-images.githubusercontent.com/45226848/231709370-207e0851-cc7b-4926-9f50-dadff20528fe.png)    |

Reply:
Hi @aamirkhan2478  
thank you for the great suggestions, <img src="https://media1.giphy.com/media/l0K4mq3yRICCW4czm/giphy.gif"/>I have taken them to mind and made changes to my code like you suggested
going to use my last code review

Reply:
- [ ] You did a good job working on the previous reviewer's suggestions 🚀, however, the error still persists:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/231774795-eccd286d-c3a8-4681-b8a6-644eb45c1a97.png) | ![image](https://user-images.githubusercontent.com/77069011/231774377-83320324-3978-4524-9454-097b0031e7e2.png) |

Please consider fixing this! 🤓 🔧